### PR TITLE
Story/quad 44

### PR DESCRIPTION
**Pull request by suprajkolluri**
Removed the QStoreUserManager classes and directly imported properties into the network manager class.

### CODE REVIEW

**Comment by jdamerow** [comment]
retest this please

File: Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkManager.java [hunk 1]
```diff
@@ -808,6 +828,9 @@ public String getNetworkXMLFromQStoreForAListOfCStatements(String xml) {
 		HttpHeaders headers = new HttpHeaders();
 		headers.setContentType(MediaType.APPLICATION_XML);
 		headers.setAccept(mediaTypes);
+		String authHeader = getAuthHeader();
+        headers.set("Authorization", authHeader);
+        
 		HttpEntity<String> request = new HttpEntity<String>(xml,headers);
 
 		try{
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkManager.java]: please correct formatting

**Comment by jdamerow** [comment]
once the formatting is corrected I'll merge it

**Comment by suprajkolluri** [comment]
Formatted the file and retested the code. I am able to connect to Qstore from Quadriga with the new authentication in place.

**Pull request merged**